rest_framework-stubs/serializers.pyi

```diff
@@ -142,7 +143,16 @@ class ListSerializer(BaseSerializer):
 
 def raise_errors_on_nested_writes(method_name: str, serializer: BaseSerializer, validated_data: Any) -> None: ...
 
+class ModelSerializerMeta:
```

Comment:
Can you please write a test for it?

Reply:
yea, sure

Reply:
what's the proper way to test this? should I just test the revealed types for the `ModelSerializer.Meta.*`?

Reply:
@sobolevn I've added the test for the above^. If you think I should do something additional, just point it out :)

Reply:
btw I was thinking, maybe it's better to define `Meta` as a class inside `ModelSerializer`, like this:
```python
class ModelSerializer(Serializer):
    class Meta:
        model: Type[Model]
        fields: Optional[Union[Sequence[str], str]]
        read_only_fields: Optional[Sequence[str]]
        exclude: Optional[Sequence[str]]
        depth: Optional[int]
        extra_kwargs: Dict[str, Dict[str, Any]]
    ...
```